src/snapred/backend/dao/request/CalibrationAssessmentRequest.py

```diff
@@ -27,7 +27,7 @@ class CalibrationAssessmentRequest(BaseModel):
     useLiteMode: bool
     focusGroup: FocusGroup
     calibrantSamplePath: str
-    workspaces: Dict[WorkspaceType, List[WorkspaceName]]
+    workspaces: Dict[WorkspaceType, WorkspaceName]
```

Comment:
Unfortunately I had to touch this part of the code.  I remember we had discussed removing the nested workspace lists, so since I was here I did that.  I can revert it if it's desired.